pkg/oomstore/push_processor.go

```diff
@@ -78,7 +85,7 @@ func (s *OomStore) InitStreamPushProcessor(ctx context.Context, cfg *types.Strea
 				processor.buffer.Range(func(key, value interface{}) bool {
 					groupID := key.(int)
 					b := value.(GroupBuffer)
-					if len(b.records) > 0 && time.Since(b.lastPush) > processor.minPeriod {
+					if len(b.records) > 0 && time.Since(b.lastPush) >= processor.flushInterval {
```

Comment:
should we use the `min(maxTickInterval, processor.flushInterval)`

Reply:
explained offline.